### DISCUSSION TITLE
Using Conversion Specifiers for the log file name with "log_file" parameter

### DISCUSSION THREAD

**Discussion by fractal815** [Motion-Project/motion#1747] [Q&A]
I want to use %Y-%m-%d in in my log file name, so that the name changes daily.   The motion.config contains /home/frank/motion/logs/motion-%Y-%m-%d.log.   The parameters are not resolved, so my file is named motion-%Y-%m-%d.log.   Why do the % parameters not resolve for log-file?

**Comment by tosiara** [comment]
I think it is not possible without changing the code. Log file needs to be "rotated", which means, flush, close the handle and open a new one. And this is not implemented at the moment. And, according to @Mr-Dave , such feature could only be implemented in motionplus (if ever decided to do so)

**Comment by pataquets** [comment]
@fractal815 I think Apache's [rotatelogs](https://httpd.apache.org/docs/2.4/programs/rotatelogs.html) tool which does exactly what you want, but that was a long time ago. Check it out if it helps and let us know.
